onnxruntime/test/onnx/main.cc

```diff
@@ -308,8 +308,11 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"matmul_2d", "disable reason"},
       {"matmul_3d", "disable reason"},
       {"matmul_4d", "disable reason"},
-      {"cast_STRING_to_FLOAT", "Cast opset 9 not supported yet"},
-      {"cast_FLOAT_to_STRING", "Cast opset 9 not supported yet"},
+      {"cast_STRING_to_FLOAT", "Linux CI has old ONNX python package with bad test data"},
```

Comment:
I think we added a mechanism to exclude from specific version of onnx? like L302

Reply:
I will explicitly disable for just 141 after a fix to a CI script to make sure the 'latest' onnx version is really installed (Linux CI build won't pass without that) is checked in soon.